roles/ceph-osd/tasks/bcache.yml

```diff
@@ -10,14 +10,14 @@
   shell: ceph osd crush add-bucket {{ hybrid_pool }} root
   register: result
   changed_when: result.stdout |search("added bucket")
-  run_once: true
+  when: inventory_hostname == (play_hosts | intersect(groups['ceph_osds_hybrid']))[0]
```

Comment:
how is this avoid re-run on first host when ceph-osd is included by some other role?  run_once will ensure task is only run once for entire play.